.github/workflows/rust.yml

```diff
@@ -104,3 +104,19 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       # `too_many_arguments` is relatively arbitrary
       - run: cargo clippy -- -A clippy::too_many_arguments -D warnings
+
+  changelog:
+    name: Changelog
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Check that changelog updated
+        uses: dangoslen/changelog-enforcer@v3
```

Comment:
This action drives me crazy, it does not work consistently with forks. I'm fine with enforcing the `CHANGELOG` but I would prefer to use something else.

Reply:
I've only found https://github.com/Zomzog/changelog-checker but it seems like is not actively developed (it still uses NodeJS12 and has a issue open for it with no action taken so far) so I will remove the check for the time being.

Reply:
Okay, thanks for checking at least.